docs/usage/shoot_maintenance.md

```diff
@@ -30,7 +30,7 @@ You can change it later, of course.
 ## Automatic Version Updates
 
 The `.spec.maintenance.autoUpdate` field in the shoot specification allows you to control how/whether automatic updates of Kubernetes patch and machine image versions are performed.
-During the maintenance, Gardener checks if there are newer versions available, and if you opt-in, then your shoot will be updated accordingly.
+Machine image version are updated per worker pool.
```

Comment:
```suggestion
Machine image versions are updated per worker pool.
```

Reply:
@danielfoehrKn could you apply this proposal?
The rest is looking good to me.

Reply:
done